src/commands/create_command.py

```diff
@@ -30,3 +33,21 @@ def validate(schema_file, schema_dir):
     def check_database(self):
         if self.data[Keys.DATABASE] is None:
             raise WrongParameterError("No database detected")
+
+    @staticmethod
+    def create_table_schema(t_path, table):
+        file = open(os.path.join(t_path,table[Keys.NAME]+"_schema.json"),'w')
+        json.dump(table, file)
+        file.close()
+
+    @staticmethod
```

Comment:
Look, I recommend to create a class called Table
‘’’
Class Table
  name 
   columns[]
   primaryKey
  Indeces[]

  Public void write () // this method write table schema, create table folder + 
     Loop for each index and index.write ()
‘’’

Class Index //
    Name
    write() // I recommend every index has its own write. Also u can make each index its own file

Reply:
Also table should have a validate method that validate the following 

- primary key is in list of column
- It should loop in each index and validate it.
- 

Index should have validate method which check the name of index exists in column table